app/scripts/directives/blocks.js

```diff
@@ -153,3 +154,11 @@ angular.module('confRegistrationWebApp')
         restrict: 'E'
       };
     });
+
+angular.module('confRegistrationWebApp')
+  .directive('birthDateQuestion', function () {
+    return {
+      templateUrl: birthDateQuestionTemplate,
```

Comment:
Ya since these templates look identical, I would just use the `dateQuestionTemplate` here. If it has some additional functionality we can create a new version later.